charter.html

```diff
@@ -80,7 +80,7 @@ <h1 id="title">PROPOSED Private Advertising Technology Working Group Charter</h1
          motivated by the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG Ethical Web Principles</a>, is to
          specify web features and APIs that support advertising while
          acting in the interests of users, in particular providing strong
-         privacy assurances.  The Working Group welcomes participation from
+         privacy assurances using technical means.  The Working Group welcomes participation from
```

Comment:
```suggestion
         privacy assurances using predominantly technical means.  The Working Group welcomes participation from
```

Reply:
@samuelweiler I'm fine with this change if you are, and then I am fine merging.

Reply:
fine with me